helm/idol-nifi/templates/nifi/hpa.yaml

```
@@ -12,14 +12,14 @@
 apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
-  name: idol-nifi
+  name: nifi
   labels: {{- include "idol-library.labels" (dict "root" . "component" .Values.nifi) | nindent 4 -}}
     app.kubernetes.io/name: nifi
 spec:
   scaleTargetRef:
     apiVersion: apps/v1
     kind: StatefulSet
-    name: idol-nifi
+    name: nifi
```

Needs to match the generated statefulset
```suggestion
    name: {{ .Values.name }}
```